src/main/resources/form_templates.json

```diff
@@ -2788,124 +2788,165 @@
   "isAuthenticationRequired" : true,
   "isFesEnabled" : true,
   "messageTextIdList": [1,4]
+}, {
+  "_id" : "RESOLUTIONS",
+  "formName" : "Resolution - Part of a share capital submission (not including reduction of capital)",
+  "formCategory" : "SH-OTHER",
+  "fee" : "",
+  "isAuthenticationRequired" : true,
+  "isFesEnabled" : true,
+  "messageTextIdList": [1,4]
```

Comment:
this is 6575, and because of issues we're going to have with 2 RESOLUTIONS I think this should go back under `CC` for now